www/index.rst

```diff
@@ -28,44 +29,59 @@ few restrictions.
 Getting Started
 ---------------
 
-- `Getting NumPy <https://www.scipy.org/scipylib/download.html>`__
-- `Installing the SciPy Stack <https://www.scipy.org/install.html>`__
-- `NumPy Tutorial <https://www.numpy.org/devdocs/user/quickstart.html>`__
-- `NumPy for MATLAB© Users <https://www.numpy.org/devdocs/user/numpy-for-matlab-users.html>`__
-- `NumPy functions by category <https://www.numpy.org/devdocs/reference/routines.html>`__
-- `NumPy Mailing List <https://mail.python.org/mailman/listinfo/numpy-discussion>`__
+To install NumPy, we strongly recommend using a
+*scientific Python distribution*.  See
+`Installing the SciPy Stack <https://www.scipy.org/install.html>`__ for
+details.
+
+Many high quality online tutorials, courses, and books are available to get
+started with NumPy. For a quick introduction to NumPy we provide the
+`NumPy Tutorial <https://www.numpy.org/devdocs/user/quickstart.html>`__. We
+also recommend the `SciPy Lecture Notes <https://scipy-lectures.org>`__ for a
+broader introduction to the scientific Python ecosystem.
 
 For more information on the SciPy Stack (for which NumPy provides the
 fundamental array data structure), see `scipy.org <https://www.scipy.org/>`__.
 
-
 Documentation
 -------------
 
-`Latest (development) version <https://www.numpy.org/devdocs>`__
-
-* `Version 1.16.1 <https://docs.scipy.org/doc/numpy-1.16.1>`__,
-  `[HTML+zip] <https://docs.scipy.org/doc/numpy-1.16.1/numpy-html-1.16.1.zip>`__
-  `[PDF] <https://docs.scipy.org/doc/numpy-1.16.1/numpy-ref-1.16.1.pdf>`__
-* `Version 1.15.4 <https://docs.scipy.org/doc/numpy-1.15.4>`__,
-  `[HTML+zip] <https://docs.scipy.org/doc/numpy-1.15.4/numpy-html-1.15.4.zip>`__
-  `[PDF] <https://docs.scipy.org/doc/numpy-1.15.4/numpy-ref-1.15.4.pdf>`__
-* `Version 1.14.5 <https://docs.scipy.org/doc/numpy-1.14.5>`__,
-  `[HTML+zip] <https://docs.scipy.org/doc/numpy-1.14.5/numpy-html-1.14.5.zip>`__,
-  `[PDF] <https://docs.scipy.org/doc/numpy-1.14.5/numpy-ref-1.14.5.pdf>`__
-* `Version 1.13.0 <https://docs.scipy.org/doc/numpy-1.13.0>`__,
-  `[HTML+zip] <https://docs.scipy.org/doc/numpy-1.13.0/numpy-html-1.13.0.zip>`__,
-  `[PDF] <https://docs.scipy.org/doc/numpy-1.13.0/numpy-ref-1.13.0.pdf>`__
+The most up-to-date NumPy documentation can be found at
+`Latest (development) version <https://www.numpy.org/devdocs>`__.  I
```

Comment:
Extra "I" at the end of this line

Reply:
.. and the link should be `/devdocs`, since we are already on the `numpy.org` site

Reply:
fixed the "I" typo.

I'd rather not use relative links, because it's not the same repo so that will break the local build and there isn't really a benefit.